CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+## [1.3.2] - 2019-03-28
```

Comment:
seems like u are releasing, this is best to do when u are merging 1.3.x -> master. It's fine if u release right away though. 
However, you should move all PI tickets to their correct Project once you know where the fix will be done.  This will help us tag JIRA tickets.